MuvrKit/MKClassifier.swift

```diff
@@ -28,6 +28,7 @@ public struct MKClassifier {
     private let neuralNet: MKForwardPropagator
     private let windowSize = 400
     private let windowStepSize = 10
+    private let scalingFactor = Float(4.0)
```

Comment:
what does scaling buy us?
at the moment everything comes from the accelerometer which means it's already scaled.
That would make sense if we use different sensors (heart rate, acc, ...) but it just improves the results somehow?